src/libs/getBrowser/index.web.js

```diff
@@ -18,13 +18,13 @@ function getBrowser() {
         temp = userAgent.match(/\b(OPR|Edge)\/(\d+)/);
```

Comment:
Looking at https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent, I don't see a case `Edge` will be defined in the userAgent, especially for Opera. Should it be removed?